### ISSUE TITLE
usage in production

### ISSUE THREAD

**Issue by markshust** [ccorcos/meteor-react-transitioner#1]
Hi Chet,

Apparently you and me are the only ones interested in meteor + react + flowrouter animations ;)

Just wanted to check to see if this was still under development or if it still needed more testing to use close to a production environment? I'm very interested in something like this.

Thanks!
Mark

**Comment by ccorcos** [comment]
Yeah, this package as a pretty big detour in figuring things out... Velocity is pretty awesome, but it introduces this problem of coordinating these async animations and making sure they don't clobber each other. Its not very easy to deal with. Native CSS transitions, on the other hand, allow you to spam the animations and the animation will change mid animation. This is much nicer AND its easier to program :+1:. On a side-note, check out the react magic-move demo. Its pretty amazing -- you can't do that with velocity.js.

As for the routers. I've given up on all these fancy routers. They introduce too much complexity for me. I'm using my ccorcos:client-side-router (something like that). Its super basic. Then I control subscriptions and everything within my components. 

If you want to do something like a push-pop animation, check [this fiddle out](https://jsfiddle.net/ccorcos/4gvtwsr9/). And maybe answer my [StackOverflow question](http://stackoverflow.com/questions/32602364/react-how-to-send-information-to-the-parent-component)? ;) 

The thing with model web apps, is they tend to have multiple nav controllers nested inside tabs. This means that your browsers linear history with worthless to you. So I just my router to format a "route" object, and use it only as an entry point to my program. When I change tabs or push / pop on the nav, I pass these route objects around, mimicing the router objects, and update the url in the browser, but dont listen to any of the route changes from the router after the first one. Everything is handled internally.
